corehq/apps/fixtures/views.py

```diff
@@ -422,3 +423,11 @@ def _return_response(code, message):
         resp_json["message"] += "%s%s%s" % (("and following " if num_unknown_groups else ""), warn_users, upload_resp.unknown_users)
 
     return HttpResponse(json.dumps(resp_json), mimetype="application/json")
+
+
+@login_and_domain_required
+def fixture_metadata(request, domain):
+    """
+    Returns list of fixtures and metadata needed for itemsets in vellum
+    """
+    return HttpResponse(json.dumps(item_lists_by_domain(domain)))
```

Comment:
if you use `json_response` you'll get proper type conversion for annoying stuff like dates that don't serialize by default as well as a properly set content type